test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -26,7 +26,7 @@
 
 public class AddCommandTest {
     private static final List<ReadOnlyPerson> EMPTY_PERSON_LIST = Collections.emptyList();
-    private static final Set<String> EMPTY_STRING_LIST = Collections.emptySet();
+    private static final Set<String> EMPTY_STRING_LIST = new HashSet<>();
```

Comment:
is this change necessary? i think the intention was to be immutable.

Reply:
Hmm, I think I did a mass replacement for this, didn't notice it changed here as well.
Anyway, since the `Person` constructor now directly sets the `tags`, thought it would be safer to ensure that the `Set` we supply is mutable from the start.
Perhaps would be better if we make `Person` hold its own mutable copy `Set` from the start, and make it add the `Tag`s in, like in `UniqueTagList`?